website/docs/d/firewall.html.markdown

```diff
@@ -90,6 +92,14 @@ A `virtual_hub` block exports the following:
 
 * `private_ip_address` - The private IP address associated with the Azure Firewall.
 
+---
+
+A `dns_settings` block exports the following:
+
+* `custom_servers` - Specifies a list of DNS Servers. It will Use Azure provided DNS servers when no `custom_servers`  provided.
+
+* `proxy_enabled` - Whether DNS proxy is enabled. It will forward DNS requests to the DNS servers when set to `true`.
+
```

Comment:
this needs updating to match the schema above?

Reply:
updated